src/monotonic/monotonic.jl

```diff
@@ -279,6 +296,33 @@ function calcTangents(::Type{TCoeffs}, x::AbstractVector{<:Number},
     return (m, Δ)
 end
 
+function calcTangents(::Type{TCoeffs}, x::AbstractVector{<:Number},
+    y :: AbstractVector{TEl}, method::AkimaMonotonicInterpolation) where {TCoeffs, TEl}
+
+    # based on Akima (1970),
+    # "A New Method of Interpolation and Smooth Curve Fitting Based on Local Procedures",
+    # doi:10.1145/321607.321609.
+
+    n = length(x)
+    Δ = Vector{TCoeffs}(undef, n-1)
+    m = Vector{TCoeffs}(undef, n)
+    for k in 1:n-1
+        Δk = (y[k+1] - y[k]) / (x[k+1] - x[k])
+        Δ[k] = Δk
+    end
+    Γ = [3Δ[1] - 2Δ[2]; 2Δ[1] - Δ[2]; Δ; 2Δ[n-1] - Δ[n-2]; 3Δ[n-1] - 2Δ[n-2]]
+    for k in 1:n
```

Comment:
```suggestion
    for k in eachindex(m)
```

Reply:
I usually use `eachindex`, too. But it has not been used here by all other `calcTangents` methods either and I simplied copied the code from one method to modify it for the Akima method.
Maybe `1:n` is used to make the code easier too read and/or because the package is designed in such a way that it does not give you any advantage in terms of performance?

Reply:
I'll change the others later. Please change this.